backend/src/reservations/reservations.controller.ts

```diff
@@ -111,4 +125,34 @@ export class ReservationsController {
         .send(`Failed to update reservation: ${error.message}`);
     }
   }
+
+  @Get('/contact-info/:id')
+  @ApiImplicitParam({
+    name: 'id',
+    required: true,
+    description: 'Event Id, for which reservations needs to be fetched',
+    type: String,
+  })
+  async getEventsContactInformation(@Res() response, @Param('id') id: number) {
```

Comment:
This is confusing.. the action name is getEventContactInformation, but you are actually returning reservations?